components/server/src/user/user-controller.ts

```diff
@@ -184,10 +184,8 @@ export class UserController {
             } catch (e) {
                 log.error("Failed to sign-in as admin with OTS Token", e);
 
-                // Default to unauthenticated, to not leak information.
-                // We do not send the error response to ensure we do not disclose information.
-                const code = e.code || 401;
-                res.sendStatus(code);
+                // Always redirect to an expired token page if there's an error
+                res.redirect("/error/expired-ots", 307);
```

Comment:
unsure if we want to try and classify different errors, i.e. expired vs. invalid. If so, we can have an error route for each one.